packages/react-components/src/components/MessageThread.tsx

```diff
@@ -681,21 +749,36 @@ export const MessageThread = (props: MessageThreadProps): JSX.Element => {
       return;
     }
     const lastMessage: ChatMessage = messagesWithId[messagesWithId.length - 1] as ChatMessage;
-
     try {
       if (
         onMessageSeen &&
         lastMessage &&
         lastMessage.messageId &&
         lastMessage.messageId !== messageIdSeenByMeRef.current
       ) {
-        await onMessageSeen(lastMessage.messageId);
+        // lastMessageList contains every participant's last sent messages
```

Comment:
As what we discussed with Chi, we should keep the previous logic in order not to hit throttling 

Lemme explain the logic a little bit again - 
1. People will only send readReceipt to the lastest message in the bottom of thread
2. We will calculate the read line of a person then assume he/she read all the messages above
3. Follow the logic #2 calculate the read count and render it in messageThread

#2 should be done either in messageThread or selector, but having it in selector will make it easier for next contextual menu work item

Reply:
Updated PR with new logic